Proposals/TrustedAppsReplacement.md

```diff
@@ -0,0 +1,397 @@
+# Trusted Apps Replacement
+
+This proposed replacement for trusted apps details splitting it into two kinds:
+ - "Signatory Control," which allows **resource owners** to control what someone has access to based on the app and identity provider (signatory) sending the token.
```

Comment:
The use of 'someone' here seems overly vague (i.e. "...to control what **_someone_** has access to..."). Should it be expanded, e.g. "...to control what **_some agent (e.g. a person, a group, a company/corporation, or an 'authorized delegate' of a person (e.g. the parent of a child, or the guardian of a senile parent)_** has access to..."?